### PR TITLE
Added expand.ambiguous

### PR DESCRIPTION
**Pull request by Aggarwal-Ayush**
Added the feature to expand ambiguous reads with multiple approved symbols even if the gene itself is an approved symbol.

### CODE REVIEW

File: R/checkGeneSymbols.R [hunk 1]
```diff
@@ -58,7 +58,8 @@ checkGeneSymbols <- function(x,
                              chromosome = NULL,
                              unmapped.as.na = TRUE,
                              map = NULL,
-                             species = "human") {
+                             species = "human", 
+                             expand.ambiguous = F) {
```
Comment (lwaldron) [inline R/checkGeneSymbols.R]: Should spell out TRUE/FALSE in full

**Review by lwaldron — CHANGES_REQUESTED** [review]
This looks very useful, just a couple things:
1. the new argument should be documented with a `@param` roxygen2 line. What does it do?
2. Would also be good to have an example in the man page, showing the difference between calling the function with this argument as TRUE vs FALSE.
3. Spell out TRUE and FALSE (I think T and F returns a note now, and is rejected from CRAN)
4. Can you also mention it in the vignette?

File: R/checkGeneSymbols.R (hunk 1 above)
Comment (Aggarwal-Ayush) [reply R/checkGeneSymbols.R]: Done.

**Comment by Aggarwal-Ayush** [comment]
> 
> 
> This looks very useful, just a couple things:
> 
>     1. the new argument should be documented with a `@param` roxygen2 line. What does it do?
> 
>     2. Would also be good to have an example in the man page, showing the difference between calling the function with this argument as TRUE vs FALSE.
> 
>     3. Spell out TRUE and FALSE (I think T and F returns a note now, and is rejected from CRAN)
> 
>     4. Can you also mention it in the vignette?

Hi @lwaldron 
I've made the required changes.

**Pull request merged**